docs/guide/data_migration/importing_data.md

```diff
@@ -174,6 +175,14 @@ When updating a segment group or segment, you can match the object to update by
 [[= include_file('code_samples/data_migration/examples/create_segment.yaml', 18, 24) =]]
 ```
 
+### Settings
+
+The following example shows how to create and update a database setting:
```

Comment:
I think it should be noted that this is an **application setting** stored in database. It's not a "database" setting per se.